packages/twenty-front/src/modules/object-record/record-show/hooks/useRecordIdsFromFindManyCacheRootQuery.ts

```diff
@@ -0,0 +1,30 @@
+import { RecordGqlRefEdge } from '@/object-record/cache/types/RecordGqlRefEdge';
+import { useApolloClient } from '@apollo/client';
+import { createApolloStoreFieldName } from '~/utils/createApolloStoreFieldName';
+
+export const useRecordIdsFromFindManyCacheRootQuery = ({
```

Comment:
regarding the comment above, here it would be find if fieldVariables was also containing the cursor